### DISCUSSION TITLE
Is there a way to determine if the open mailbox is a shared mailbox?

### DISCUSSION THREAD

**Discussion by mailchecktool-admin** [OfficeDev/office-js#4904] [Q&A]
Is there an API to determine if a mailbox in Outlook is a shared mailbox? Also, is it possible to distinguish whether the mailbox is in the user's email compose window or opened in a separate tab?

**Comment by alex-fitzgerald** [comment]
AFAIK - and I'd love to know if there are other ways - the only guaranteed way to know if you're in a shared mailbox is if the `getSharedPropertiesAsync` method is available on an email, i.e.:

```javascript
// own inbox, item(s) selected
window.Office.context.mailbox.item.getSharedPropertiesAsync() // undefined

// shared inbox, item(s) selected
window.Office.context.mailbox.item.getSharedPropertiesAsync() // defined
```

Note however that in all clients except Classic Desktop this method appears to be undefined it multiple items are selected even in a shared inbox.

**Comment by mailchecktool-admin** [comment]
@alex-fitzgerald 
Thank you for your response.
I will verify if it is available!
